src/components/logo/Logo.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../variables";
+
+*{
+	box-sizing: border-box;
+}
+
+body {
+	min-height: 450px;
+	height: 100vh;
+	margin: 0;
+	background: #e7ece8;
+}
+
+$i: 1;
+$NUM_LAYERS: 20;
+@for $i from 1 through $NUM_LAYERS {
+	.layer:nth-child(#{$i}):after{
+		transform: translateZ(($i - 1) * -1.5px);
+	}
+}
+
+.stage {
+	height: 50px;
+	width: 70px;
+	margin: auto;
+	position: relative;
+	top: 0; right: 0; bottom: 0; left: 0;
+	perspective: 9999px;
+	transform-style: preserve-3d;
+	.layer {
+		width: 100%;
+		height: 100%;
+		position: absolute;
+		transform-style: preserve-3d;
+		animation: ಠ_ಠ 5s infinite alternate ease-in-out -7.5s;
+		animation-fill-mode: forwards;
+		transform: rotateY(40deg) rotateX(33deg) translateZ(0);
+		&:after{
+			font: 70px/0.65 'Pacifico', 'Kaushan Script', Futura, 'Roboto', 'Trebuchet MS', Helvetica, sans-serif;
+			content: 'aH!';
+			white-space: pre;
+			text-align: left;
+			height: 100%;
+			width: 100%;
+			position: absolute;
+			color: lighten(color(white), 4%); // #fff
+			letter-spacing: -2px;
+			text-shadow: 4px 0 10px hsla(0, 0%, 0%, .13);
+		}
+		&:nth-child(n+#{round($NUM_LAYERS/2)}):after {
+			-webkit-text-stroke: 3px hsla(0, 0%, 0%, .25);
+		}
+		&:nth-child(n+#{round($NUM_LAYERS/2 + 1)}):after {
+			-webkit-text-stroke: 15px color(primary); // dodgerblue
+			text-shadow: 6px 0 6px darken(color(primary),35%),
+			5px 5px 5px darken(color(primary),40%),
+			0 6px 6px darken(color(primary),35%);
+		}
+		&:nth-child(n+#{round($NUM_LAYERS/2 + 2)}):after {
+			-webkit-text-stroke: 15px darken(color(primary), 10%);
+		}
+		&:last-child:after {
+			-webkit-text-stroke: 17px hsla(0, 0%, 0%, .1);
+		}
+
+	}
+}
+
+//
```

Comment:
Consider removing all the commented out code in case you don't need it.

Reply:
@Estaer , we are still resolving the issues.

Reply:
Cool